docs/sources/awss3.md

```diff
@@ -0,0 +1,160 @@
+# Event Source for Amazon S3
+
+This event source subscribes to event notifications from an [Amazon S3 bucket][s3-docs]. Events are published by S3 to
+an [Amazon SQS queue][sqs-docs] in order to be consumable by the event source.
+
+## Prerequisites
+
+### S3 Bucket
+
+If you didn't already do so, create a S3 bucket by following the instructions at [Create your first S3
+bucket][s3-create].
+
+### Amazon Resource Name (ARN)
+
+A fully qualified ARN is required to uniquely identify the Amazon S3 bucket.
+
+The ARN displayed in the AWS Console has the format `arn:aws:s3:::{bucketName}`. This ARN unfortunately lacks two
+essential informations: the [AWS region][aws-region] and the [account ID][aws-acc-id], which must both be included into
+the ARN by using the more complete format below:
+
+```
+arn:aws:s3:{awsRegion}:{awsAccountId}:{bucketName}
+```
+
+![S3 Bucket ARN](../images/awss3-source/arn-region-1.png)
+
+### API Credentials
+
+The TriggerMesh event source for Amazon S3 authenticates calls to the AWS API using an [Access Key][accesskey]. The page
+at this link contains instructions to create an access key when signed either as the root user or as an IAM user. Take
```

Comment:
The page at which link ? is that referring to the previous link ?

Reply:
You're right that's some pretty poor phrasing here. I'll change this sentence.

Reply:
@sebgoa rephrased, and updated all other docs pages for AWS event sources. Could you please take another look?